docs/Config_Reference.md

```diff
@@ -4824,6 +4824,50 @@ vssa_pin:
 #   noise. The default is 2 seconds.
 ```
 
+### [ads1x1x]
+
+ADS1013, ADS1014, ADS1015, ADS1113, ADS1114 and ADS1115 are I2C based Analog to
+Digital Converters that can be used for temperature sensors. They provide 4
+analog input pins either as single line or as differential input.
+
+Note: Use caution if using this sensor to control heaters. The heater min_temp
+and max_temp are only verified in the host and only if the host is running and
+operating normally. (ADC inputs directly connected to the micro-controller
+verify min_temp and max_temp within the micro-controller and do not require a
+working connection to the host.)
+
+The sample rate of the chip gets automatically calculated based on the number of
+pins used and the refresh rate requested by klipper.
```

Comment:
FWIW, seems a little odd to describe parameters the user doesn't need to configure - it might be confusing.